cmake/Init.cmake

```diff
@@ -7,3 +7,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+
+if("${FPC_PATH} " STREQUAL " ")
```

Comment:
As part of build there shouldn't be really a requirement to have FPC_PATH env defined, we should be able to determine that via TOP?  Also note that in doc we do not really require FPC_PATH to be defined, we just informally use it to characterize examples and alike ..

that said, i notice that it seems Marcus already "sinned" in the last fabric build PR which also requires now FPC_PATH.  If we want to rely on it, i think we should define it in `config.mk`.   Relying on a env variable is also a bit brittle in case you have multiple installations on same machine (e.g., one with concept release 1 and another one with master), in which case you might get to confusing situation if your make reads partially stuff with the TOP (i.e., what you expect) and partially stuff via FPC_PATH (which would be the wrong place ...)

Reply:
I suppose I've sinned too!
The `FPC_PATH` variable is frequently handy, and clearly there is problem with `TOP`.
I would vote to keep the first and fix the second -- of course, through an issue and separate PR.
@mbrandenburger , your opinion?

Reply:
what is the problem with TOP? But in any case, why not just define FPC_PATH in `config.mk` as suggested?  It fixes the issue by ensuring that the build (a) has the (not really documented) value and (b) it is consistent?

Reply:
The problem with TOP, as you mentioned, is consistency -- it should be the same as FPC_PATH.

I am ok with `config.mk`.

Oh you mean initializing `FPC_PATH` from `TOP`. I was thinking the opposite.
But, again as you said, I think we already need `FPC_PATH` defined somewhere else. So would this work out-of-the-box?

Reply:
right now build fails if you do not have FPC_PATH defined. With above change to `config.mk` FPC_PATH should be defined properly during build, regardless whether you had it originally in your environment or not.

Reply:
will fix it -- I thought it was a precondition already!

Reply:
Having FPC_PATH in config.mk makes sense for me! :) I like it

Reply:
* FPC_PATH is defined in config.mk
* Init.cmake checks that it exists
@g2flyer  would you double check again that this is fine?

Reply:
That's what we discussed above and is consistent with the change to SGX_SSL and the other half of related changes from #338.   I actually thought, this would be already in an earlier commit; if not, even better than it is now here :-)

Reply:
I had removed it completely but... I still needed to define the cmake-internal FPC_PATH from the one defined in the environment.
Anyway... solved!